app/Models/Dashboard.php

```diff
@@ -72,9 +72,9 @@ public function widgets()
      * @param $user_id
      * @return Builder|static
      */
-    public function scopeAllAvailable(Builder $query, $user_id)
```

Comment:
Should have mentioned it, I know it shouldn't be part of this pr but the tests break without this change as the original code was wrong.